docs/topics/files.txt

```diff
@@ -239,3 +240,11 @@ For example::
 
     class MyModel(models.Model):
         my_file = models.FileField(storage=select_storage)
+
+In order to set a storage defined in the :setting:`STORAGES` setting use lambda
+function::
```

Comment:
I think this is a useful example (even if we might want to revisit needing a lambda later). Thanks. 

```suggestion
In order to set a storage defined in the :setting:`STORAGES` setting you can
use lambda function::
```